server/modules/modules.go

```diff
@@ -45,88 +47,124 @@ func Message(message *messageService.Message, messageType int, conn *websocket.C
 		}
 	}
 }
+func sendMes(conn *websocket.Conn, newHeader messageService.MessageHeader) {
+	newMessage := messageService.Message{
+		Header: newHeader,
+	}
+	marshaledMessage, err := messageService.MarshalMessage(&newMessage)
+	if err != nil {
+		loger.Log.Errorf("Can`t marshal message. %s", err)
+		conn.Close()
+		return
+	}
+	if err := conn.WriteMessage(websocket.TextMessage, marshaledMessage); err != nil {
+		loger.Log.Errorf("Can not send message. %s", err)
+		conn.Close()
+		return
+	}
 
+}
 func Register(message *messageService.Message, conn *websocket.Conn) {
 	if _, ok := customers.Clients[message.Header.UserName]; ok {
 		loger.Log.Warn("User already exist")
+		err := errors.New("User already exist")
+		newMessageHeader := messageService.MessageHeader{
+			Type_:    coremessage.RegisterType,
+			Command:  err.Error(),
+			UserName: "",
```

Comment:
You can simply skip empty fields they will be filled with zero values on compilation